docs/components/concepts/clusters.md

```diff
@@ -6,72 +6,88 @@ description: "Learn more about the clusters available in your Camunda 8 plan."
+<p><strong>** Management Components</strong> means the components responsible for designing and making processes and decisions executable, such as Web Modeler and Console.</p>
+<p><strong>***  RTO (Recovery Time Objective)</strong> means the maximum allowable time that a system or application can be down after a failure or disaster before it must be restored. It defines the target time to get the system back up and running. <strong>RPO (Recovery Point Objective)</strong> means the maximum acceptable amount of data loss measured in time. It indicates the point in time to which data must be restored to resume normal operations after a failure. It defines how much data you can afford to lose. The RTO/RPO figures shown in the table are provided on a best-effort basis and are not guaranteed.</p>
 
-- Clusters tagged as `test`, `stage`, or `prod` do not auto-pause.
-- Paused clusters are automatically deleted after 30 consecutive paused days. You can change the tag to avoid cluster deletion.
-- No data is lost while a cluster is paused. All execution and configuration is saved, but cluster components such as Zeebe and Operate are temporarily disabled until you resume the cluster.
+:::info
+See [Camunda Enterprise General Terms](https://legal.camunda.com/licensing-and-other-legal-terms#camunda-enterprise-general-terms) for term definitions for **Monthly Uptime Percentage** and **Downtime**.
+:::
 
-:::tip
+## Cluster size
 
-To prevent auto-pause, you can:
+The cluster size defines the cluster performance and capacity.
 
-- Tag the cluster as `test`, `stage`, or `prod` instead of `dev`.
-- [Upgrade your Free Trial plan](https://camunda.com/pricing/) to a Starter, Professional, or Enterprise plan.
+- You can choose from three sizes of cluster (1x, 2x, 3x).
+- Each increase in size boosts cluster performance and adds capacity. Larger cluster sizes allow you to serve more workload.
+- Increased usage such as higher throughput or longer data retention requires a larger cluster size. See [sizing your environment](/components/best-practices/architecture/sizing-your-environment.md).
+- Each size increase uses one of your available cluster reservations.
 
-:::
+### Cluster size hardware packages
 
-## Development clusters
+Cluster sizes determine the hosting hardware package used for the cluster, as follows:
 
-Development clusters, available in the Starter and Enterprise plans, are recommended for development, testing, proof of concepts, and demos.
+| Size  | 1x  | 2x  | 3x  |
+| :---- | :-- | :-- | :-- |
+| Spec? |     |     |     |
 
-The way this type of cluster works varies depending on if you are using it in the Starter or the Enterprise plan.
+### Limitations
 
-### Development clusters in the Enterprise Plan
+- Currently, there are limits on the number of packages you can select.
```

Comment:
I believe the maximum is 4 then, maybe we should be specific

Reply:
Done - added into note